migrate/v0_13/migrate_test.go

```diff
@@ -43,17 +45,35 @@ func TestMigrateCdp(t *testing.T) {
 
 }
 
-func TestMigrateAuth(t *testing.T) {
+func TestAuth(t *testing.T) {
 	bz, err := ioutil.ReadFile(filepath.Join("testdata", "kava-4-auth-state-block-500000.json"))
 	require.NoError(t, err)
 	var oldGenState auth.GenesisState
 	cdc := app.MakeCodec()
 	require.NotPanics(t, func() {
 		cdc.MustUnmarshalJSON(bz, &oldGenState)
 	})
-	newGenState := MigrateAuth(oldGenState)
+	newGenState := Auth(oldGenState)
 	err = auth.ValidateGenesis(newGenState)
 	require.NoError(t, err)
 	require.Equal(t, len(oldGenState.Accounts), len(newGenState.Accounts)+1)
 
 }
+
+func TestCommittee(t *testing.T) {
+	bz, err := ioutil.ReadFile(filepath.Join("testdata", "kava-4-committee-state.json"))
+	require.NoError(t, err)
+	var oldGenState v0_11committee.GenesisState
+	cdc := codec.New()
+	sdk.RegisterCodec(cdc)
+	v0_11committee.RegisterCodec(cdc)
+	require.NotPanics(t, func() {
+		cdc.MustUnmarshalJSON(bz, &oldGenState)
+	})
+
+	newGenState := Committee(oldGenState)
+	err = newGenState.Validate()
+	require.NoError(t, err)
+
+	require.Equal(t, len(oldGenState.Committees), len(newGenState.Committees))
+}
```

Comment:
nit: hacky test to check no permissions were dropped
```suggestion
	require.Equal(t, len(oldGenState.Committees), len(newGenState.Committees))
	
	for i := 0; i < len(oldGenState.Committees); i++ {
		require.Equal(t, len(oldGenState.Committees[i].Permissions), len(newGenState.Committees[i].Permissions))
	}
	
	oldSPCP := oldGenState.Committees[0].Permissions[0].(v0_11committee.SubParamChangePermission)
	newSPCP := newGenState.Committees[0].Permissions[0].(v0_13committee.SubParamChangePermission)
	require.Equal(t, len(oldSPCP.AllowedParams), len(newSPCP.AllowedParams))
	require.Equal(t, len(oldSPCP.AllowedAssetParams), len(newSPCP.AllowedAssetParams))
	require.Equal(t, len(oldSPCP.AllowedCollateralParams), len(newSPCP.AllowedCollateralParams))
	require.Equal(t, len(oldSPCP.AllowedMarkets), len(newSPCP.AllowedMarkets))
}
```